views/components/Graphs/BarGraph.jsx

```diff
@@ -13,7 +13,18 @@ import {
 import { graphStyles } from '../../styles/chart_styles';
 
 const BarGraph = ({ graph }) => {
-  const { targetValuesMap } = graph
+  const transformDataToPercentage = (dataset) => {
+    const totals = dataset[0]?.map((data, i) => {
+      return dataset.reduce((memo, curr) => {
+        return memo + curr[i].count;
```

Comment:
I think we want to use the target if all data has a target and use the count if we're missing any targets.